tests/v2/test_status_v2_sync.py

```diff
@@ -66,6 +70,75 @@ def status_handler(request: Request):
                     content_type="application/json")
 
 
+def filesystem_handler(request: Request):
```

Comment:
I don't have a strong opinion but should we split the tests in different files? Otherwise we can just rename the file